eventmesh-connectors/eventmesh-connector-spring/src/main/java/org/apache/eventmesh/connector/spring/source/MessageSendingOperations.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.connector.spring.source;
+
+import org.apache.eventmesh.common.enums.ProtocolType;
+
+/**
+ * Operations for sending messages to a destination.
+ */
+public interface MessageSendingOperations {
```

Comment:
It has been completed in a new round of submissions.

Reply:
> > I am still confused about the Source you have implemented. You start the Source Connector Server by launching it through the Spring container, but after your Source is started, the `queue` of `SpringSourceConnector` is always empty, and I didn't see any behavior of obtaining data from a source in any form during the startup process. Could you explain it?
> > 
> > 我对你实现的Source仍然有困惑。你通过Spring容器启动来启动Source Connector Server，但是你的Source启动完成后，`SpringSourceConnector`的`queue`一直是空的，在启动过程中我好像没看到从某个源以某种形式获取数据的行为。 你能否解答一下？
> > > Since the source side is the user system itself, the user needs to actively send the message to the queue by calling the send method of the `SpringSourceConnector`
> > > > Are you sure this is the design of the Source Connector in the community? This means that the continuous invocation of the poll method on the source after runtime startup will be limited by the user's call to `MessageSendingOperations#send()`.
> > > > 
> > > > 你确定这是社区对于Source Connnector的设计吗？这样runtime启动以后对source的poll方法的不断调用要受限于用户对`MessageSendingOperations#send()`的手动调用。
> 
> @xwm1992 Can the Source Connector be designed like this now?

spring source connector poll message from the memory queue is ok, for spring application user need send the message to the local queue for source connector poll.